templates/gunicorn.conf

```diff
@@ -1,8 +1,7 @@
 [program:{{ project_name }}-server]
-command={{ source_dir }}/dotenv.sh {% if use_newrelic %}{{ venv_dir }}/bin/newrelic-admin run-program {% endif %}{{ venv_dir }}/bin/gunicorn {{ project_name }}.wsgi:application --bind=0.0.0.0:{{ project_port }} --workers={{ gunicorn_num_workers }} {% if gunicorn_num_threads is defined %}--threads={{ gunicorn_num_threads }}{% endif %}
-
```

Comment:
Perhaps it has to do with jinja's handling of whitespace around the ``{% if %}`` tag at the end of the line, since ``celery.conf`` doesn't seem to have the extra newline.

Reply:
Good catch @copelco. I've put the blank line back, along with a comment so we don't forget and remove it again.